### PR TITLE
explicit is now photon

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ By default, the build is targeted against Eclipse Oxygen / 4.7.
 You can explicitly set the `eclipse.target` property to
```
Comment (chanseokoh) [inline README.md]: Above this line is "By default, the build is target against Eclipse Oxygen / 4.7."

That line should also be changed to "... against Eclipse Photon / 4.8."

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ By default, the build is targeted against Eclipse Oxygen / 4.7.
...
 `photon` (4.8).
 ```
-$ mvn -Declipse.target=oxygen package
+$ mvn -Declipse.target=photon package
```
Comment (chanseokoh) [inline README.md]: I noticed this still doesn't work. I think it should be

`$ (cd eclipse ; mvn -Pide-target-platform -Declipse.target=XXX package)`

Right, @briandealwis?

File: README.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ By default, the build is targeted against Eclipse Oxygen / 4.7.
 You can explicitly set the `eclipse.target` property to
 `photon` (4.8).
```
Comment (briandealwis) [inline README.md]: This line should be changed to:
> `oxygen` (4.7) or `2018-09` (4.9)

File: README.md (hunk 2 above)
Comment (briandealwis) [reply README.md]: This should remain at `eclipse.target=oxygen`.  `photon` doesn't work as it's now the default; Maven doesn't allow an OR-style profile activation unfortunately.

Comment (chanseokoh) [reply README.md]: I also think you need `-Pide-target-platform` after moving into the `eclipse` sub-directory.

Comment (briandealwis) [reply README.md]: Yup, @chanseokoh's right.

File: README.md (hunk 1 above)
Comment (elharo) [reply README.md]: done

**Pull request merged**